Add note about authentication middleware (#91)
OpsLevel/backstage-plugin: README.md

If you're running Self-Hosted OpsLevel, replace `target` with your URL.

> > :warning: **If you are using an authentication solution with Backstage**: Ensure that it does not interfere with the HTTP headers going through this proxy. If requests to OpsLevel seemingly inexplicably fail with a HTTP 401 (Unauthorized) response, but you have confirmed that your API token is valid, this is one possible root cause. Due to the customizable nature of Backstage, we can provide limited support in such cases.

Reviewer: YMMV: I feel like this copy is accidentally too positive at the end: "if it's broken, we'll provide support! (but only a little)" instead of "this problem is complicated, we won't be able to help very much if you're doing this"

```suggestion
> :warning: **If you are using an authentication solution with Backstage**: Ensure that it does not interfere with the HTTP headers going through this proxy. If requests to OpsLevel seemingly inexplicably fail with a HTTP 401 (Unauthorized) response, but you have confirmed that your API token is valid, this is one possible root cause. Due to the customizable nature of Backstage, the support we will be able to offer in such cases will be limited.
```
